HISTORY.rst

```diff
@@ -4,6 +4,16 @@ Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`__.
 
+[41.0.0] - 2021-09-01
```

Comment:
We can update this date to when we actually create the release. (I'm not sure how accurate this needs to be, but I've tended to try to get it accurate.)